spec/cc/cli/analyze_spec.rb

```diff
@@ -2,6 +2,21 @@
 
 module CC::CLI
   describe Analyze do
+    describe '#initialize' do
+      it 'sets dev mode for --dev' do
+        instance = Analyze.new(['--dev'])
+
+
+        assert instance.dev_mode?, "in dev mode"
```

Comment:
We use the `must_` syntax - can you reformat these lines? Should be `instance.dev_mode?.must_equal true`